pkgs/applications/audio/cozy-audiobooks/default.nix

```diff
@@ -0,0 +1,86 @@
+{ stdenv, fetchFromGitHub
+, ninja
+, boost
+, meson
+, pkgconfig
+, wrapGAppsHook
+, appstream-glib
+, desktop-file-utils
+, gtk3
+, glib
+, gst_all_1
+, gobjectIntrospection
+, python3Packages
+, file
+, cairo
+, sqlite
+, gettext
+, gnome3
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
`buildPythonApplication` seems to be better for this. See [gnome-music](https://github.com/NixOS/nixpkgs/blob/master/pkgs/desktops/gnome-3/apps/gnome-music/default.nix).

Reply:
i don't see how this would work as there is no `setup.py`

Reply:
`buildPythonApplication` takes a parameter `format` which should be set to `other` in this case

https://github.com/NixOS/nixpkgs/blob/29d0eec15faa315b45bbae08f43ac40e7cb2b4aa/pkgs/desktops/gnome-3/apps/gnome-music/default.nix#L11

Reply:
okay, i changed that. who knew!